create_lmodsitepackage.py

```diff
@@ -248,51 +248,6 @@
     return accel_stack_in_modulepath
 end
 
-local function eessi_removed_module_warning_startup_hook(usrCmd)
```

Comment:
Removing this seems wrong, still used in `eessi_startup_hook`?

Also, if we really want to remove this, can we do that in a dedicated PR?